lib/outdated.js

```diff
@@ -87,6 +87,8 @@ class Outdated extends ArboristWorkspaceCmd {
     // sorts list alphabetically
     const outdated = this.list.sort((a, b) => a.name.localeCompare(b.name, 'en'))
 
+    process.exitCode = outdated.length ? 1 : 0
```

Comment:
There is already an outdated.length check below that returns if it's `0` length.  We can set the `exitCode` to `1` after that, since we'll have returned otherwise.

Reply:
It's an `&&` condition, so it could be that `this.npm.config.get('json') === true`